x/rollapp/types/params.go

```diff
@@ -72,6 +118,39 @@ func (p Params) String() string {
 	return string(out)
 }
 
+func validateLivenessSlashBlocks(i interface{}) error {
```

Comment:
`interface{}` => `any`

Reply:
all the others are the same
you can make an issue to change all of these to `any` across the codebase